src/integration-tests/tests/driver.rs

```diff
@@ -88,6 +90,18 @@ mod driver {
             .map_err(report)
     }
 
+    #[test_case(sql::client::SqlInstancesService::builder(); "default")]
+    #[test_case(sql::client::SqlInstancesService::builder().with_tracing(); "with tracing enabled")]
+    #[test_case(sql::client::SqlInstancesService::builder().with_retry_policy(retry_policy()); "with retry enabled")]
```

Comment:
I think you probably want a combination of `with_tracing().with_retry_policy(...)` the motivation for having multiple tests is not very strong for this library.